Library/Formula/sdf2-bundle.rb

```diff
@@ -0,0 +1,37 @@
+class Sdf2Bundle < Formula
+  desc "parser and parser generator for syntax definition formalism (SDF2)"
+  homepage "http://www.program-transformation.org/Sdf/SdfBundle"
+  url "http://releases.strategoxt.org/strategoxt-0.17/sdf2-bundle/sdf2-bundle-2.4pre212034-37nm9z7p/sdf2-bundle-2.4.tar.gz"
+  sha256 "2ec83151173378f48a3326e905d11049d094bf9f0c7cff781bc2fce0f3afbc11"
+
+  depends_on "aterm"
+  depends_on "pkg-config" => :build
+
+  conflicts_with "sdf", :because => "Both install multiple common libraries and includes"
```

Comment:
Here is where I specify the conflict with `sdf`. I have added a similar `conflicts_with` block (and a `revision`) to `sdf`'s formula, but have not pushed that commit yet, because it currently fails `brew audit --strict` due to lacking `test do`